step-capstone/src/App.js

```diff
@@ -1,12 +1,49 @@
 import React from 'react';
 import './styles/App.css';
-import TravelObject from './components/TravelObject'
+import Trip from "./components/Trip"
+import TripItemComponent from "./components/TripItemComponent"
 
 class App extends React.Component {
-  render () {
-    return(
+  constructor() {
+    super();
+    
+    this.state = {
+      selectedTrip: undefined
+    };
+
+    this.handleOpenTrip = this.handleOpenTrip.bind(this);
+  }
+
+  handleOpenTrip(tripID) {
+    // fetch trip from database with tripID
+    this.setState({ selectedTrip: "fetched trip" })
+    console.log("Opening trip " + tripID);
+    // redirect to trip component
```

Comment:
I think this is something you intend to do later. For comments like that the convention is to put `TODO ({your username}): {whatever it is you'll do later}`. That makes it easy to search through the code for stuff you need to finish and for your teammates to understand what's up. You do this in a few places but not consistently.